docs/networking.md

```diff
@@ -9,9 +9,13 @@ help you out.
 
 
 {% include /style/icon.html type="info" %}
-For Raspberry Pi platforms with no display, you must use the wired Ethernet port
-to connect for the first time. You can set up additional connections using the
-`connmanctl` command through a remote SSH session.
+For Raspberry Pi platforms with no display, you must use the 
+wired Ethernet port to connect for the first time through a remote SSH 
+session. An alternative method is to connect the pi to an HDMI screen monitor 
+and USB keyboard and press <kbd>Alt</kbd>+<kbd>F2</kbd> at the end of
+the boot sequence to reach a login prompt (id: robot, password: maker). Once 
+connected, you can then set up additional connections using 
+[`connmanctl`](https://wiki.archlinux.org/index.php/Connman#Usage) commands.
```

Comment:
Due to the way our CSS works, links which are code-formatted don't turn out very well, and code formatting in alerts is also odd. If you change it to 

```
[connmanctl](https://wiki.archlinux.org/index.php/Connman#Usage){: .alert-link }
```
it'll format it as a proper link in the alert, but without code formatting.